articles/cross-origin-authentication/index.md

```diff
@@ -10,9 +9,9 @@ contentType:
 useCase: 
   - strategize
 ---
-# Cross-Origin Authentication
+# Cross-origin Authentication
```

Comment:
In APA and in our other writing afaik, hyphenation in Title Case is done with capitalization on the second word as well (Cross-Origin).